### PR TITLE
FEATURE: TMA-221 Ruby SDK supports URN for date dimension

### PR DESCRIPTION
**Pull request by vhtien**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Changelog
- [ ] Build passing

### CODE REVIEW

**Comment by vhtien** [comment]
Documentation: https://github.com/gooddata/gooddata-ruby-examples/pull/40

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/736/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/690/ : FAILURE in 3m 10s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/461/ : FAILURE in 15m 13s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/737/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/691/ : SUCCESS in 46m 36s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/462/ : UNSTABLE in 16m 19s

File: spec/integration/urn_date_dim_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      p.add_date_dimension('created_on', urn: 'urn:pe:date')
+    end
+  end
+
+  after(:all) do
+    @client && @client.disconnect
+  end
+
+  context 'project creates from blueprint contains urn in date dimension' do
+    before do
```
Comment (panjan) [inline spec/integration/urn_date_dim_spec.rb]: Please use `before :all` and `after :all` so that we don't recreate the project every time.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/740/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/694/ : SUCCESS in 44m 26s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/465/ : FAILURE in 26s

**Comment by panjan** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/762/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/717/ : SUCCESS in 46m 52s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/487/ : SUCCESS in 20m 13s

**Pull request merged**